### ISSUE TITLE
add `require-schema-suffix` rule

### ISSUE THREAD

**Issue by marcalexiei** [marcalexiei/eslint-plugin-zod-x#13]
Allow user to define which zod import source should be used across the codebase:

### Options

N/A

### Configuration 

```ts
export default defineConfig({
  // ...
  rules: {
   'zod/require-schema-suffix': ['error']
  }
  // ...
})
```

### Valid 

```ts
const string = z.string();
```

### Invalid

```ts
const string = z.string();
```

If user apply the proposed fix the result will be:

```ts
const stringSchema = z.string();
```
